notebooks/263-stable-diffusion-torchdynamo-backend/263-stable-diffusion-torchdynamo-backend.ipynb

```diff
@@ -0,0 +1,420 @@
+{
```

Comment:
<p> I believe these functions required only for gradio? There is no need any extra wrappers for running pipeline, right?</p><p>I think it is important point to show how simple to use torch.compile that nothing changed from original pipeline call and only one line of code added. </p><p>Using wrappers for call pipeline can may weaken this effect and reduce readability.</p><p> </p><p>If it is used only for gradio, can it be moved closed to gradio and use pipeline directly for the places?  </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/1338/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>